lib/rucio/common/config.py

```diff
@@ -166,30 +166,30 @@ def config_get(
         convert_type_fnc: 'Callable[[str], _T]' = lambda x: x,
 ) -> Union[_T, _U]:
     """
-    Return the string value for a given option in a section
-
-    First it looks at the configuration file and, if it is not found, check in the config table only if it is called
-    from a server/daemon (and if check_config_table is set).
-
-    :param section: the named section.
-    :param option: the named option.
-    :param raise_exception: Boolean to raise or not NoOptionError, NoSectionError or RuntimeError.
-    :param default: the default value if not found.
-    :param clean_cached: Deletes the cached config singleton instance if no config value is found
-    :param check_config_table: if not set, avoid looking at config table even if it is called from server/daemon
-    :param session: The database session in use. Only used if not found in config file and if it is called from
-                    server/daemon
-    :param use_cache: Boolean if the cache should be used. Only used if not found in config file and if it is called
-                      from server/daemon
-    :param expiration_time: Time after that the cached value gets ignored. Only used if not found in config file and if
-                            it is called from server/daemon
-    :param convert_type_fnc: A function used to parse the string config value into the desired destination type
-
-    :returns: the configuration value.
-
-    :raises NoOptionError
-    :raises NoSectionError
-    :raises RuntimeError
+    Return the configuration value for a given option in a section.
+
+    First, the configuration file is checked. If the option is not found,
+    and if called from a server/daemon (with check_config_table enabled),
+    the configuration table is queried.
+
+    Parameters:
+      section: the named section.
+      option: the named option.
+      raise_exception: whether to raise an exception (NoOptionError, NoSectionError, or RuntimeError)
+                       if the option is not found. When a non-None default is provided, this parameter is redundant.
+      default: the value to return if the option is not found. If no value is desired, omit it.
+      clean_cached: clears the cached configuration instance if no value is found.
+      check_config_table: indicates whether to check the configuration table if the option is missing.
+      session: the database session to use (only applicable for server/daemon calls).
+      use_cache: whether to use caching for server/daemon calls.
+      expiration_time: the cache expiration time in seconds.
+      convert_type_fnc: a function used to convert the retrieved configuration value.
```

Comment:
Old version was clearer, as it clarifies that the retrieved configuration value is always a string, and that's why a conversion function might be needed for certain cases.